tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -3174,7 +3205,94 @@ def lwaTileAssignment(self, kernel, tP):
       with self.allocTmpSgpr(1) as tmpSgprInfo:
         module.add(vectorStaticRemainder(dummy, dividendReg, "Serial", kernel["WavefrontSize"], tmpVgprRes, tmpSgprInfo))
 
-    module.add(vectorStaticDivideAndRemainder(qReg, rReg, dividendReg, divisor, tmpVgprRes))
+    if isDTVAB:
```

Comment:
here produces a lot of redundant codes.
Could we store the required tile assignment in LraTileAssignmentMFMA for DTV case?

Reply:
Originally, this function was graTileAssignment (Tensile still uses graTileAssignment).
This part of calculation is for both global read and local write.
I can create a sub function like graTileAssignmentDTV and call it from here.
I do not think it is appropriate to put this part of code in LraTileAssignmentMFMA .
I need to copy code related to global read to LraTileAssignmentMFMA.

Reply:
What I can come up with is
- Add a sub function for LraTileAssignmentMFMA (code generation part only)
- Call the sub function from here (pass src and dest vgprs as parameters, plus isDTVAB)
- Add conditions for DTV

Is that something you expect?
I would like to have your comment before I start changing this.
I need to refactor LRA code.

Reply:
Please keep in mind that majority of the code will be shared but this will make LRA code more complicated.

Reply:
@jichangjichang 
I unified the DTV tile assignment code with LraTileAssignmentMFMA.
Please take a look at the new change.